gps/identifier.go

```diff
@@ -20,17 +20,17 @@ import (
 // management domain has lots of different path-ish strings floating around:
 //
 //  actual directories:
-//	/home/sdboyer/go/src/github.com/sdboyer/gps/example
+//	/home/sdboyer/go/src/github.com/golang/dep/gps/example
```

Comment:
As with the deduce tests, this is really talking about the meaning of project roots. So, either all these docs changes should be reverted, or become s/`sdboyer/gps`/`golang/dep`/